packages/zowe-explorer/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### Bug fixes
 
+- The "Zowe Resources" panel is now hidden by default until it is used to display a table or other data. [#3113](https://github.com/zowe/zowe-explorer-vscode/issues/3113)
```

Comment:
Can we reword this to say: " ... until the user opens the panel to display a table or other data." I'm trying to use active voice here.

Reply:
Zowe Explorer will open the panel automatically when displaying a table or other data. That said, I will update the changelog to clarify this.

Reply:
Addressed in 5830d76